lib/al/Library/Screen/ScreenFunction.cpp

```diff
@@ -0,0 +1,37 @@
+#include <al/Library/Screen/ScreenFunction.h>
```

Comment:
Libraries don't need to use <> for their own headers and this file is in the same folder as this header so it can simply be `#include "ScreenFunction.h"

Reply:
This is fine, it's more consistent if al always uses angle brackets and full paths for its includes